kahuna/public/stylesheets/main.css

```diff
@@ -942,15 +944,19 @@ input.search-query__input {
     height: 36px;
 }
 
-/* TODO: remove this once eveything is in the action-bar */
-.results-count {
-    float: left;
-    padding: 0 10px;
+.related-labels {
+    line-height: 35px;
```

Comment:
Annoyingly not, line-height with px / united values is poorly supported or adhered to.